.github/workflows/check-modules.yml

```diff
@@ -0,0 +1,14 @@
+name: check-modules
+on: [pull_request]
```

Comment:
I added a [few types of events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) but turned out to duplicate the execution of the event (`[opened, closed, reopened, synchronize, ready_for_review ]`). So far, not specifying them seem to work better.